packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -278,7 +277,7 @@ export class InMemoryAnchorService implements AnchorService, AnchorValidator {
       //TODO (NET-1657): Update the InMemoryAnchorService to mirror the behavior of the contract-based anchoring system
       txType: V1_PROOF_TYPE,
     }
-    txnCache.set(txHashCid.toString(), timestamp)
+    TRANSACTION_CACHE.set(txHashCid.toString(), timestamp)
```

Comment:
Can we inject rather than import this?

Reply:
Would love to, but again, please not for now. "Too many changes." Let me track that separately.